src/Synapse/Synapse/help/Enable-AzSynapseWorkspace.md

```diff
@@ -8,7 +8,11 @@ schema: 2.0.0
 # Enable-AzSynapseWorkspace
 
 ## SYNOPSIS
-Activate a new Azure Synapse Analytics workspace.
+When creating an Azure Synapse Analytics workspace, you can choose to encrypt all data at rest in the workspace 
+with a customer-managed key which will provide double encryption to the workspace.
+You may need to set up the encryption environment firstly, such as to create a key vault with purge protection enable and 
+specify Access Polices to the key vault.
+Then use this cmdlet to activate the new Azure Synapse Analytics workspace which double encryption is enabled using a customer-managed key.
```

Comment:
Put the description in one line? The file will be eventually transformed to a web page of Microsoft Docs. I suppose the website can handle the word wrapping.

Reply:
The description itself looks great to me.

Reply:
Sure, I will put the words in one line.